Tryouts/Prototypes/Shell/Node-launcher/Lib/package.json

```diff
@@ -9,6 +9,9 @@
     "clean": "rimraf output",
     "build": "npm run clean && tsc && rollup -c"
   },
+  "bin": {
+    "composeui-cli": "./cli/cli.js"
```

Comment:
Can we be consistent with naming? "composeui" or "compose", just use the same naming everywhere.

Reply:
There is an existing compose-cli for something else (https://github.com/docker/compose-cli) so I picked this name instead.

Reply:
That's just the `docker-compose` command. I'm fine with `composeui` but then shouldn't we rename the other packages as well, like `@morgan-stanley/composeui-messaging-client`, `@morgan-stanley/composeui-node-launcher`, etc?

Reply:
.NET namespaces are already prefixed with `MorganStanley.ComposeUI`

Reply:
https://www.npmjs.com/package/compose-cli

Reply:
No repo, no website, no scope, last published 2 years ago, I think you can ignore that.

Reply:
BTW our CLI command could be just `composeui`.